.github/workflows/gitlab_ci.yml

```diff
@@ -7,7 +7,7 @@ on:
   pull_request_target:
     types: [opened, synchronize, reopened, labeled]
   schedule:
-    - cron: '1 5 * * 1'
+    - cron: '2 5 * * 1'
```

Comment:
Does this mean you want to run more or less often? Why?

Reply:
Neither, I want to start it a minute later. By this, the other workflow is started a minute earlier than this one, meaning that querying the other workflow works as it is already started